Plugins/Public/mobiledocking_plugin/MobileDocking Plugin.vc14.vcxproj

```diff
@@ -14,13 +14,13 @@
     <ProjectGuid>{81A35B95-1CDD-4F58-A24C-E2DC47D43DD0}</ProjectGuid>
     <RootNamespace>MobileDocking Plugin</RootNamespace>
     <Keyword>Win32Proj</Keyword>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.20348.0</WindowsTargetPlatformVersion>
     <ProjectName>MobileDocking Plugin</ProjectName>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v143</PlatformToolset>

```

Comment:
Is 143 standardised now? All plugins should be the same regardless

Reply:
Yes, entire DiscoHook is now compiled in VS2022